chamge/amqp/amqp-rpc.h

```diff
@@ -0,0 +1,58 @@
+/**
+ *  Copyright 2019 SK Telecom, Co., Ltd.
+ *    Author: Heekyoung Seo <hkseo@sk.com>
+ *
+ */
+
+#ifndef __CHAMGE_AMQP_RPC_H__
+#define __CHAMGE_AMQP_RPC_H__
+
+#include <gio/gio.h>
+#include <chamge/types.h>
+
+G_BEGIN_DECLS
+#define CHAMGE_TYPE_AMQP_RPC       (chamge_amqp_rpc_get_type ())
+    CHAMGE_API_EXPORT
```

Comment:
I guess this object is only for internal use. then, we don’t need to export these apis.

Reply:
Yes.